src/pages/index.js

```diff
@@ -93,28 +220,52 @@ addButton.addEventListener("click", handleOpenAddImageModal);
 /* -------------------------------------------------------------------------- */
 
 const previewImagePopup = new PopupWithImage("#preview-image-modal");
+
+const deleteImageConfirmPopup = new PopupWithFormConfirmDelete(
+  "#confirm-modal",
+  handleDeleteImageSubmit
+);
+
+function handleDeleteImageSubmit(data) {
+  api.removeImageFromAPI("cards", data.imageId);
```

Comment:
You should remove the card and close the popup only in the case of the successful api response